virttest/libvirt_xml/network_xml.py

```diff
@@ -732,15 +732,14 @@ def state_dict(self):
         if self.defined:
             return self.virsh.net_state_dict(virsh_instance=self.virsh)[self.name]
 
-    def create(self):
+    def create(self, **dargs):
         """
         Adds non-persistant / transient network to libvirt with net-create
         """
-        cmd_result = self.virsh.net_create(self.xml)
-        if cmd_result.exit_status:
-            raise xcepts.LibvirtXMLError("Failed to create transient network %s.\n"
-                                         "Detail: %s" %
-                                         (self.name, cmd_result.stderr))
+        try:
+            self.virsh.net_create(self.xml, **dargs)
```

Comment:
this is against with previous implementation where one exception will be thrown out depending on status although virsh.net_create normally executed.